.github/workflows/release.yml

```diff
@@ -13,6 +13,8 @@ jobs:
     runs-on: ubuntu-18.04
     steps:
       - uses: actions/checkout@master
+      - name: Abort for dev version
```

Comment:
It didn't actually work to prevent the release. But fortunately .dev* versions count as prerelease and our code to check whether there's an upgrade available only includes final releases...so no real harm done.